cni/go.mod

```diff
@@ -0,0 +1,19 @@
+module kuma.io/cni
+
+go 1.13
```

Comment:
1.18.

What is the reason we have a separate go module for CNI? So far, we had a pretty bad experience with nested go modules and we migrated away from this model.